packages/devtools_app/test/inspector/inspector_error_navigator_test.dart

```diff
@@ -79,6 +81,8 @@ void main() {
 
     testWidgets(
       'can navigate backwards',
+      // Intebtionally unwaited.
```

Comment:
```suggestion
      // Intentionally unwaited.
```

Reply:
I think you could also just make the closure async
`(WidgetTester tester) async => testNavigate(`